packages/material-ui-envinfo/package.json

```diff
@@ -6,7 +6,7 @@
   },
   "license": "MIT",
   "description": "Logs infos about the environment relevant to @mui/*",
-  "version": "1.1.6",
+  "version": "1.1.7",
```

Comment:
Maybe we should to a minor to signal the change of name of the package? 
```suggestion
  "version": "1.2.0",
```

Reply:
no strong opinion, waiting for other team member's thought before changing it.

Reply:
I would even do a major, to be honest, but minor sounds good too. Definitely not a patch.

Reply:
let's make a major change then. @eps1lon what's your opinion?

Reply:
Breaking change for safe meassure.

Reply:
changed to `2.0.0`